gremlin-go/driver/traversal.go

```diff
@@ -450,3 +450,29 @@ func (tp *textP) And(args ...interface{}) TextPredicate {
 func (tp *textP) Or(args ...interface{}) TextPredicate {
 	return newTextPWithP("or", *tp, args...)
 }
+
+type withOptions struct {
+	Tokens  string
+	None    int32
+	Ids     int32
+	Labels  int32
+	Keys    int32
+	Values  int32
+	All     int32
+	Indexer string
+	List    int32
+	Map     int32
+}
+
+var WithOptions withOptions = withOptions{
+	Tokens:  "~tinkerpop.valueMap.tokens",
+	None:    0,
+	Ids:     1,
```

Comment:
Shall we change the translator as well, so it is covered by tests?

Reply:
All other GLV's use these values directly so I'd prefer to leave it since it will be a bunch of extra work that no other drivers did.